### ISSUE TITLE
Multiqc Kallisto module ZeroDivisionError

### ISSUE THREAD

**Issue by TarjaTH** [ewels/MultiQC#1746]
I apologise if this is trivial, but I checked other entries with this kind of error and I did not find a solution.

I am attaching the file that triggered the error. It is true that there are some entries in the file that have 0 reads pseudoaligned (including the first entry), but not all of them. I am aware, that I generally have a low level of alignment, I am still trouble shooting the sequencing protocol, but I would still like to visualise the results as I have many files and need an overview and your tool seemed perfect for that.

Before, I had an error about the file size limit, but I found another entry describing this error, so I increased the limit in the config_defaults_yaml file.

Thank you very much.

```console
$ multiqc -v .

  /// MultiQC 🔍 | v1.13.dev0

[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  This is MultiQC v1.13.dev0
[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  Command used: /home/tarja/anaconda3/bin/multiqc -v .
[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  Latest MultiQC version is v1.12
[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  Working dir : /home/tarja/Dcoronatus/20220630_P2022-097-NES/mapping/multiqc
[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  Template    : default
[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  Running Python 3.8.8 (default, Apr 13 2021, 19:58:26)  [GCC 7.3.0]
[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  Analysing modules: custom_content, ccs, ngsderive, purple, conpair, lima, peddy, somalier, methylQA, mosdepth, phantompeakqualtools, qualimap, preseq, quast, qorts, rna_seqc, rockhopper, rsem, rseqc, busco, bustools, goleft_indexcov, gffcompare, disambiguate, supernova, deeptools, sargasso, verifybamid, mirtrace, happy, mirtop, sambamba, homer, hops, macs2, theta2, snpeff, gatk, htseq, bcftools, featureCounts, fgbio, dragen, dedup, pbmarkdup, damageprofiler, biobambam2, jcvi, mtnucratio, picard, vep, sentieon, prokka, qc3C, nanostat, samblaster, samtools, sexdeterrmine, eigenstratdatabasetools, bamtools, jellyfish, vcftools, longranger, stacks, varscan2, snippy, bbmap, bismark, biscuit, hicexplorer, hicup, hicpro, salmon, kallisto, slamdunk, star, hisat2, tophat, bowtie2, bowtie1, snpsplit, odgi, pangolin, kat, leehom, adapterRemoval, clipandmerge, cutadapt, flexbar, kaiju, kraken, malt, trimmomatic, sickle, skewer, sortmerna, biobloomtools, fastq_screen, afterqc, fastp, fastqc, pychopper, pycoqc, minionqc, multivcfanalyzer, clusterflow, checkqc, bcl2fastq, bclconvert, interop, ivar, flash, seqyclean, optitype, whatshap
[2022-08-16 11:40:46] multiqc                                            [DEBUG  ]  Using temporary directory for creating report: /tmp/tmpvpl8z8ri
[2022-08-16 11:40:46] multiqc                                            [INFO   ]  Search path : /home/tarja/Dcoronatus/20220630_P2022-097-NES/mapping/multiqc
|         searching | ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 1773/1773  
[2022-08-16 11:40:56] multiqc                                            [DEBUG  ]  Summary of files that were skipped by the search: [skipped_file_contents_search_errors: 91956] // [skipped_module_specific_max_filesize: 3000] // [skipped_no_match: 1755]
[2022-08-16 11:40:56] multiqc.plots.bargraph                             [DEBUG  ]  Using matplotlib version 3.3.4
[2022-08-16 11:40:56] multiqc.plots.linegraph                            [DEBUG  ]  Using matplotlib version 3.3.4
[2022-08-16 11:40:56] multiqc                                            [DEBUG  ]  No samples found: custom_content
╭────────────────────────── Oops! The 'kallisto' MultiQC module broke... ──────────────────────────╮
│ Please copy this log and report it at https://github.com/ewels/MultiQC/issues                    │
│ Please attach a file that triggers the error. The last file found was:                           │
│ ./kallisto_mapping_Dcor-1_scaff1_21.log                                                          │
│                                                                                                  │
│ Traceback (most recent call last):                                                               │
│   File "/home/tarja/anaconda3/lib/python3.8/site-packages/multiqc/multiqc.py", line 654, in run  │
│     output = mod()                                                                               │
│   File "/home/tarja/anaconda3/lib/python3.8/site-packages/multiqc/modules/kallisto/kallisto.py", │
│     self.parse_kallisto_log(f)                                                                   │
│   File "/home/tarja/anaconda3/lib/python3.8/site-packages/multiqc/modules/kallisto/kallisto.py", │
│     "percent_aligned": (paligned_reads / total_reads) * 100,                                     │
│ ZeroDivisionError: float division by zero                                                        │
│                                                                                                  │
╰──────────────────────────────────────────────────────────────────────────────────────────────────╯
[2022-08-16 11:40:56] multiqc                                            [DEBUG  ]  Oops! The 'kallisto' MultiQC module broke...
================================================================================
Traceback (most recent call last):
  File "/home/tarja/anaconda3/lib/python3.8/site-packages/multiqc/multiqc.py", line 654, in run
    output = mod()
  File "/home/tarja/anaconda3/lib/python3.8/site-packages/multiqc/modules/kallisto/kallisto.py", line 36, in __init__
    self.parse_kallisto_log(f)
  File "/home/tarja/anaconda3/lib/python3.8/site-packages/multiqc/modules/kallisto/kallisto.py", line 84, in parse_kallisto_log
    "percent_aligned": (paligned_reads / total_reads) * 100,
ZeroDivisionError: float division by zero
================================================================================
[2022-08-16 11:40:56] multiqc                                            [WARNING]  No analysis results found. Cleaning up..
[kallisto_mapping_Dcor-1_scaff1_21.log](https://github.com/ewels/MultiQC/files/9349628/kallisto_mapping_Dcor-1_scaff1_21.log)
```

**Comment by ewels** [comment]
Hi @TarjaTH,

Is it possible to attach an example file that I can use to reproduce the error please?

Phil

**Comment by ewels** [comment]
x-ref https://github.com/ewels/MultiQC/issues/1429

**Comment by TarjaTH** [comment]
Yes, of course, thank you. I am sorry, I thought this was attached. I hope it is working now.

[kallisto_mapping_Dcor-1_scaff1_21.log](https://github.com/ewels/MultiQC/files/9349815/kallisto_mapping_Dcor-1_scaff1_21.log)

**Comment by ewels** [comment]
Thanks @TarjaTH - fixed in 50487e8edc1e85d2c609efd945a6d34c4b774325 . Should be available in a stable release imminently.
